__tests__/chat.test.ts

```diff
@@ -1,28 +1,37 @@
 import crypto from 'crypto'
 import fs from 'fs'
 import Bot from '../lib'
-import config from './tests.config.js'
+import config from './tests.config'
 import {timeout} from '../lib/utils'
 import {pollFor} from './test-utils'
 import {promisify} from 'util'
+import {ChatChannel, ReadResult, MessageSummary} from '../lib/chat-client/types'
+import {OnMessage} from '../lib/chat-client'
 
-test('Chat methods with an uninitialized bot', () => {
+test('Chat methods with an uninitialized bot', (): void => {
```

Comment:
Does TS give an error if we don't include the `: void` type signature on functions like these? Would of thought that it could infer that 🤷‍♂

Reply:
something was mad and made me add it...maybe eslint?